integration/noderunner/noderunner_test.go

```diff
@@ -60,9 +60,12 @@ func TestCanStartStandaloneObscuroHostAndEnclave(t *testing.T) {
 		t.Fatal(err)
 	}
 	network := gethnetwork.NewGethNetwork(int(gethPort), int(gethWebsocketPort), gethBinaryPath, 1, 1, []string{address.String()})
-	defer network.StopNodes()
 	go enclaverunner.RunEnclave(enclaveConfig)
 	go hostrunner.RunHost(hostConfig)
+	obscuroClient := obscuroclient.NewClient(clientServerAddr)
+
+	defer network.StopNodes()
```

Comment:
I think best to leave this on the line below the network creation, seems to be the done thing to pair creation and tear-down for clarity and makes sure it gets cleaned up if NewClient() panics for example.

Reply:
👍

Reply:
@BedrockSquirrel I remembered - the reason I did it this way is that it makes more sense to stop the Obscuro side first, and _then_ the L1 side, since L1 is the server and Obscuro the client in this relationship.

Reply:
Ah ignore, it goes in the other direction, right? FIFO?

Reply:
@joeldudley Yeah it's LIFO, so they're executed in reverse order. But you haven't changed the order of the deferrals by moving the network.StopNodes up, before my comment the L1 teardown deferral was before the L2 deferral and it still is (so both then and now it tears down L2 first).